ui/pages/first-time-flow/metametrics-opt-in/metametrics-opt-in.component.js

```diff
@@ -0,0 +1,162 @@
+                <div className="metametrics-opt-in__row-description">
+                  {t('metametricsCommitmentsNeverCollectIP', [
+                    <span
+                      className="metametrics-opt-in__bold"
+                      key="neverCollectIP"
+                    >
+                      {t('metametricsCommitmentsBoldNever')}
+                    </span>,
+                  ])}
+                </div>
+              </div>
+              <div className="metametrics-opt-in__row">
+                <i className="fa fa-times" />
+                <div className="metametrics-opt-in__row-description">
+                  {t('metametricsCommitmentsNeverSellDataForProfit', [
+                    <span
+                      className="metametrics-opt-in__bold"
+                      key="neverSellData"
+                    >
+                      {t('metametricsCommitmentsBoldNever')}
+                    </span>,
+                  ])}
+                </div>
+              </div>
+            </div>
+          </div>
+          <div className="metametrics-opt-in__footer">
+            <PageContainerFooter
+              onCancel={async () => {
+                await setParticipateInMetaMetrics(false);
+
+                history.push(INITIALIZE_SELECT_ACTION_ROUTE);
+              }}
+              cancelText={t('noThanks')}
+              hideCancel={false}
+              onSubmit={async () => {
+                await setParticipateInMetaMetrics(true);
+                try {
+                  if (
+                    participateInMetaMetrics === null ||
+                    participateInMetaMetrics === false
+                  ) {
+                    await trackEvent(
+                      {
+                        category: EVENT.CATEGORIES.ONBOARDING,
+                        event: EVENT_NAMES.METRICS_OPT_IN,
+                        properties: {
+                          action: 'Metrics Option',
+                          legacy_event: true,
+                        },
+                      },
+                      {
+                        isOptIn: true,
+                        flushImmediately: true,
+                      },
+                    );
+                  }
+                } finally {
+                  history.push(INITIALIZE_SELECT_ACTION_ROUTE);
+                }
+              }}
+              submitText={t('affirmAgree')}
+              disabled={false}
+            />
+            <div className="metametrics-opt-in__bottom-text">
+              {t('gdprMessage', [
+                <a
+                  key="metametrics-bottom-text-wrapper"
+                  href="https://metamask.io/privacy.html"
+                  target="_blank"
+                  rel="noopener noreferrer"
+                >
+                  {t('gdprMessagePrivacyPolicy')}
+                </a>,
+              ])}
+            </div>
+          </div>
+        </div>
+      </div>
+    );
+  }
+}
```

Comment:
You shouldn't be adding an entire file here? Maybe rebase mistake?

Reply:
I merged the latest develop upstream branch into my branch, after that it shows that I created 2 files. So that is confusing me. I am having a hard time identifying my mistake.

Reply:
I use rebase because I think it is cleaner but there shouldn't be a problem using merge. I'm clocking off for the week but will try to look at this next week

Reply:
And @georgewrmarshall this file that I worked on
<img width="693" alt="image" src="https://user-images.githubusercontent.com/53999998/218238848-57f07964-03ce-4d09-b182-aafe394a07bb.png">
does not exist in the latest develop branch. Please suggest a suitable action for this as well.

Reply:
@georgewrmarshall is this file redundant? Should I remove it from my branch?

Reply:
Looks like it! I think you can go ahead and remove.